apps/test/unit/templates/sectionsRefresh/SectionsSetUpContainerTest.jsx

```diff
@@ -31,6 +31,21 @@ describe('SectionsSetUpContainer', () => {
     ).to.equal('Finish creating sections');
   });
 
+  it('renders edit header and save button', () => {
+    const wrapper = shallow(
+      <SectionsSetUpContainer sectionToBeEdited={true} />
```

Comment:
I'm sure you're working on this part still, but just a visual reminder to edit now that the input is no longer a boolean!